adoc/admin-velero-prereqs.adoc

```diff
@@ -12,10 +12,10 @@ Add {suse} helm chart repository URL:
 helm repo add suse https://kubernetes-charts.suse.com
 ----
 
-=== Object Storage And It's Credentials
+=== Object Storage And Its Credentials
 
 Velero uses object storage to store backups and associated artifacts.
-It can also optionally create snapshots of persistent volume and store them in object storage by `restic`, if there is no supported volume snapshot provider.
+It can also optionally create snapshots of persistent volume and store them in object storage by `restic` if there is no supported volume snapshot provider.
```

Comment:
```suggestion
It can also optionally create snapshots of persistent volumes and store them in object storage via `restic` if there is no supported volume snapshot provider.
```